google/cloud/logging_v2/handlers/__init__.py

```diff
@@ -16,12 +16,14 @@
 
 from google.cloud.logging_v2.handlers.app_engine import AppEngineHandler
 from google.cloud.logging_v2.handlers.container_engine import ContainerEngineHandler
+from google.cloud.logging_v2.handlers.structured_log import StructuredLogHandler
 from google.cloud.logging_v2.handlers.handlers import CloudLoggingHandler
 from google.cloud.logging_v2.handlers.handlers import setup_logging
 
 __all__ = [
     "AppEngineHandler",
     "CloudLoggingHandler",
     "ContainerEngineHandler",
+    "StructuredLogHandler",
     "setup_logging",
```

Comment:
Worth adding a comment to explain semantics of functionality that each handler adds.

Reply:
I don't believe it's standard to use `__init__.py` for documentation.This is just an internal file used for importing packages. It's likely to get out of date if things change in the future, so I'd prefer to keep documentation alongside code